peps/pep-0798.rst

```diff
@@ -700,6 +713,65 @@ PEP.  As such, these forms should continue to raise a ``SyntaxError``, but with
 a new error message as described above, though it should not be ruled out as a
 consideration for future proposals.
 
+.. _pep798-alternativegenexpsemantics:
+
+Alternative Generator Expression Semantics
+------------------------------------------
+
+Another point of discussion centered around the semantics of unpacking in
+generator expressions, particularly the relationship between the semantics of
+synchronous and asynchronous generator expressions given that the latter do not
+support ``yield from`` (see the section of :pep:`525` on Asynchronous ``yield
+from``).
+
+Several reasonable options were considered, none of which was a clear winner in
+a `poll in the Discourse thread
+<https://discuss.python.org/t/pep-798-unpacking-in-comprehensions/99435/33>`__:
```

Comment:
At least my vote to disallow this in async comprehensions was a clear loser :)

Reply:
:)